core/amber/src/main/scala/edu/uci/ics/texera/web/model/websocket/request/WorkflowExecuteRequest.scala

```diff
@@ -15,7 +15,8 @@ case class WorkflowExecuteRequest(
     engineVersion: String,
     logicalPlan: LogicalPlanPojo,
     replayFromExecution: Option[ReplayExecutionInfo], // contains execution Id, interaction Id.
-    workflowSettings: WorkflowSettings
+    workflowSettings: WorkflowSettings,
+    emailNotificationConfig: EmailNotificationConfig
```

Comment:
Do we really need to pass EmailNotificationConfig from the frontend to backend?
I believe the base URL(it's just workflow id), workflow name, and user email can be fetched from the backend.

Reply:
I have made the changes to let the backend fetch the information. Can you take a look?

Reply:
LGTM!